src/gui/org/deidentifier/arx/gui/resources/Resources.java

```diff
@@ -286,6 +287,20 @@ public InputStream getStream(final String name) {
     
     /**
      * Returns an image.
+     * 
+     * @param type
+     * @return
+     */
+    public Image getImage(AttributeType type) {

```

Comment:
We return in each if, do you want that else for readability?

    public Image getImage(AttributeType type) {
        if (type == AttributeType.IDENTIFYING_ATTRIBUTE) { return getManagedImage("bullet_red.png"); }
        if (type == AttributeType.INSENSITIVE_ATTRIBUTE) { return getManagedImage("bullet_green.png"); }
        if (type == AttributeType.SENSITIVE_ATTRIBUTE) { return getManagedImage("bullet_purple.png"); }
        if (type == AttributeType.QUASI_IDENTIFYING_ATTRIBUTE) { return getManagedImage("bullet_yellow.png"); }
        throw new IllegalArgumentException("Unknown attribute type '" + type +"'");
    }